docs/installing.md

```diff
@@ -116,3 +116,26 @@ Open a file called `main.bicep` VS code. If the extension is installed, you shou
 Now that you have the tooling installed, you can start the tutorial which will teach you full bicep capabilities:
 
 [1 - Working with a basic bicep file](./tutorial/01-simple-template.md)
+
+
+<br>
```

Comment:
I wanted to leave some space after the "next steps" so that people who wouldn't know or care about nightly builds don't need to be bother with it. I also thought about adding so many `<br>`s that most users wouldn't even notice that this next section exists, but I'm not sure how evil that is :)

Reply:
Why not make it a separate page and link to it so there's a boundary?

Reply:
that...makes more sense